src/interfaces/alteration-interfaces/resolutions-interface.ts

```diff
@@ -2,5 +2,6 @@
 export interface ResolutionsIF {
   date: string,
   id: number,
-  type: string
+  type: string,
+  confirmed:boolean,
```

Comment:
I think we should remove "confirmed" here and use "resolutionConfirmed" in "CreateResolutionIF". What you had in the previous commit was the correct. When we save draft, we will remove "resolutionConfirmed" from `buildSpecialResolutionFiling`: `specialResolution: {... `. This will ensure the schema is correct and enable validation for the checkbox (with additional small work).